cirq/google/gate_sets_test.py

```diff
@@ -33,7 +33,7 @@ def test_serialize_exp_w(gate, axis_half_turns, half_turns):
     q = cirq.GridQubit(1, 2)
     expected = {
         'gate': {
-            'id': 'exp_w'
+            'id': 'xy'
```

Comment:
I guess this change is ok because no service is consuming these new v2 protos?

Reply:
Correct, nothing should be using XMON that I know of.  It looks like there are some devices using this, but none are hooked up to cirq_server, so this should be fine to make backwards incompatible changes.